src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1165.mdx

```diff
@@ -0,0 +1,26 @@
+---
+subject: Infrastructure agent
+releaseDate: '2021-04-21'
+version: 1.16.5
+---
+
+## Notes
+A new version of the agent has been released. Follow standard procedures to [update the Infrastructure agent](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/update-or-uninstall/update-infrastructure-agent).
+
+New Relic recommends that you upgrade the agent regularly and at a minimum every 3 months.
+
+## Fixed
+
+* Log forwarder (FluentBit) stopped working on some Linux distro versions (Ubuntu16, RHEL7, AmazonLinux2). It complained on agent log like:
```

Comment:
Hi @varas - what do you mean with "it complained on agent log like"?

Reply:
The "log forwarder" component threw some errors through the agent log containing below messages.